Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroGraphics/Driver/MicroGraphics.cs

```diff
@@ -1351,10 +1351,19 @@ void Fill(int x, int y, int width, int height, Color color)
         {
             if (IgnoreOutOfBoundsPixels)
             {
+                if (x >= Width || 
```

Comment:
In this case, would we skip the whole fill just because the start or bounds exceed the allowable area? Or should we try to fill what we can of any union with the allowable area?

Reply:
if x is greater than the width then everything in the fill is off screen so we can safely just return